Section 2/2.15_17_Working_with_Strings_Part1_&_Part2.rb

```diff
@@ -0,0 +1,76 @@
+# WORKING WITH STRINGS Part 1
+# String concatenation
+first_name = "hanyi"
+last_name = "koh"
+puts first_name + " " + last_name
+full_name = first_name + last_name
+
+# String interpolation
+puts "My first name is #{first_name} and my last name is #{last_name}"
+
+# Using method
+puts full_name.class    
+# => String
+p 10.methods # find all the invokable methods for the object by default
+# => [:bit_length, :digits, :|, :gcd, :lcm, ...]
+p 10.class
+# => String
+## When using two methods on one object in the same line, is called method chain for example: 10.to_s.class
+p 10.to_s.class
+# => String
```

Comment:
?????

Reply:
p 10.class returns Integer
However, we can call to_s function to convert it from the Integer class to String class. After it get converted to String class then the 10.to_s.class is actually equal to '10'.class and We will be able to use the string class function like gsub as well.

The two or more methods using on the same object in the same like is called method chain.

Reply:
oh wait. you are right. i missed the `to_s`